src/mapping/RadialBasisFctMapping.hpp

```diff
@@ -145,8 +171,17 @@ void RadialBasisFctMapping<RADIAL_BASIS_FUNCTION_T>::computeMapping()
       globalOutMesh.addMesh(*outMesh);
     }
 
-    _rbfSolver = RadialBasisFctSolver<RADIAL_BASIS_FUNCTION_T>{this->_basisFunction, globalInMesh, boost::irange<Eigen::Index>(0, globalInMesh.vertices().size()),
-                                                               globalOutMesh, boost::irange<Eigen::Index>(0, globalOutMesh.vertices().size()), this->_deadAxis, _polynomial};
+    if (_useEigen) {
+      _rbfSolver = RadialBasisFctSolver<RADIAL_BASIS_FUNCTION_T>{this->_basisFunction, globalInMesh, boost::irange<Eigen::Index>(0, globalInMesh.vertices().size()),
+                                                                 globalOutMesh, boost::irange<Eigen::Index>(0, globalOutMesh.vertices().size()), this->_deadAxis, _polynomial};
+    }
+#ifndef PRECICE_NO_GINKGO
+    else {
+      _ginkgoRbfSolver.~GinkgoRadialBasisFctSolver();
```

Comment:
Why do we need to destroy the object here first? In the above version, we override the former object, which implicitly calls the destructor..

Reply:
It produced some error. Will later elaborate on that one.